server/publicapi/items/service.py

```diff
@@ -73,27 +73,27 @@ def get(self, req, lookup):
             req = ParsedRequest()
 
         allowed_params = (
-            'q', 'start_date', 'end_date',
+            'q', 'start_date', 'end_date', 'filter',
             'include_fields', 'exclude_fields'
         )
         self._check_for_unknown_params(req, whitelist=allowed_params)
 
-        # set the "q" filter
         request_params = req.args or {}
-        query_filter = {}
+        req.args = {}
 
+        # set the search query
         if 'q' in request_params:
-            # TODO: add validation for the "q" parameter when we define its
-            # format and implement the corresponding actions
-            query_filter = json.loads(request_params['q'])
+            req.args['q'] = request_params['q']
+            req.args['default_operator'] = 'OR'
+
+        if 'filter' in request_params:
+            req.args['filters'] = [{'term': json.loads(request_params['filter'])}]
```

Comment:
Yes, initially we planned for JSON, but then steered away from it.... well, actually we haven't finalized the format for the `q` parameter initially, thus we left it as a dummy parameter until we decide how it was going  to work.